moped-database/metadata/tables.yaml

```diff
@@ -3546,41 +3596,58 @@
           columns: '*'
         update:
           columns:
-            - capitally_funded
-            - is_deleted
-            - end_date
-            - added_by
-            - milestone_id
-            - project_id
-            - project_importance
-            - project_length
-            - project_order
             - project_sponsor
-            - timeline_id
             - task_order
+            - parent_project_id
+            - interim_project_id
+            - work_assignment_id
             - ecapris_subproject_id
+            - project_uuid
             - fiscal_year
-            - knack_project_id
             - project_description
-            - project_description_public
             - project_name
-            - project_priority
+            - project_id
+            - capitally_funded
             - project_website
-            - work_assignment_id
+            - end_date
+            - knack_project_id
             - date_added
-            - updated_at
-            - project_uuid
-            - parent_project_id
+            - is_deleted
+            - added_by
+            - project_lead_id
       retry_conf:
         interval_sec: 10
         num_retries: 0
         timeout_sec: 60
-      webhook_from_env: MOPED_API_EVENTS_URL
+      webhook_from_env: HASURA_ENDPOINT
       headers:
-        - name: MOPED_API_APIKEY
-          value_from_env: MOPED_API_APIKEY
-        - name: MOPED_API_EVENT_NAME
-          value: activity_log
+        - name: x-hasura-admin-secret
+          value_from_env: HASURA_GRAPHQL_ADMIN_SECRET
+      request_transform:
+        body:
+          action: transform
+          template: |-
+            {
+                            "query": "mutation InsertActivity($object: moped_activity_log_insert_input!, $project_id:Int!, $updated_at:timestamptz ) { insert_moped_activity_log_one(object: $object) { activity_id } update_moped_project_by_pk(pk_columns: {project_id: $project_id}, _set: {updated_at: $updated_at}) { updated_at }}",
```

Comment:
the mutation is all on one line, scroll far to the right to see the updated_at mutation